packages/mobile/src/hooks/useContentNodeImage.ts

```diff
@@ -28,7 +28,11 @@ const createImageSourceWithSizes = ({
     return {
       width,
       height: width,
-      uri: createUri(size)
+      uri: createUri(size),
+      // A CID is a unique identifier of a piece of content,
+      // so we can always rely on a cached value
+      // https://reactnative.dev/docs/images#cache-control-ios-only
```

Comment:
ios only? 👀
is the bug also on android?

Reply:
I think we need to build something better here long term (sounds like @sliptype has some progress on it too!).

I believe that react native image does pretty intelligently cache, but since this flag exists, I figure worth turning on for iOS?
https://rangle.io/blog/image-caching-in-react-native/

Reply:
Confirmed that the image does load offline on android, so the caching must be present in some fashion. Can't guarantee this always, though I guess